.github/workflows/pipelines.yml

```
@@ -230,115 +233,40 @@ jobs:
-          bucketName: ${{ secrets.GCP_BUCKETNAME }}
-          objectName: cuemon.snk
-
-      - name: Build
-        uses: codebeltnet/dotnet-build@v2
-        with:
-          uploadBuildArtifact: false
-
-      - name: Finalize CodeQL SAST Analysis
-        uses: codebeltnet/codeql-scan-finalize@v1
+    name: call-codeql
+    needs: [build,test]
```

YAMLlint (1.37.1)

[warning] too few spaces after comma

(commas)